simple-bills-gui/src/utils/objectUtils.ts

```diff
@@ -0,0 +1,3 @@
+export function setToZeroIfNull(value: number) {
```

Comment:
W sumie używasz to tylko w jednym miejscu

Reply:
done